datalad/support/tests/test_network.py

```diff
@@ -272,8 +275,13 @@ def test_url_samples():
     """
     $> ssh example.com/path/sp1:fname
     ssh: Could not resolve hostname example.com/path/sp1:fname: Name or service not known
+
+    edit 20190516 yoh: but this looks like a perfectly valid path.
+    SSH knows that it is not a path but its SSHRI so it can stay dumb.
+    We are trying to be smart and choose between RIs (even when we know that
+    it is e.g. a file).
     """
-    _check_ri('example.com/path/sp1:fname', SSHRI, hostname='example.com/path/sp1', path='fname')
+    # _check_ri('example.com/path/sp1:fname', SSHRI, hostname='example.com/path/sp1', path='fname')
```

Comment:
actually I will adjust here to check explicitly for being a PathRI, and bring back empty line above...